src/navigation/reroute.ts

```diff
@@ -244,18 +258,19 @@ export function reroute(
         /* We load and bootstrap apps while other apps are unmounting, but we
          * wait to mount the app until all apps are finishing unmounting
          */
-        const loadThenMountPromises = appsToLoad.map((app) => {
-          return toLoadPromise(app).then((app) =>
-            tryToBootstrapAndMount(app, unmountAllPromise)
-          );
-        });
+        const loadThenMountPromises: Promise<InternalApplication>[] =
+          appsToLoad.map((app) => {
+            return toLoadPromise(app).then((app) =>
+              tryToBootstrapAndMount(app, unmountAllPromise)
+            );
+          });
 
         /* These are the apps that are already bootstrapped and just need
          * to be mounted. They each wait for all unmounting apps to finish up
          * before they mount.
          */
-        const mountPromises = appsToMount
-          .filter((appToMount) => appsToLoad.indexOf(appToMount) < 0)
+        const mountPromises: Promise<InternalApplication>[] = appsToMount
+          .filter((appToMount) => !appsToLoad.includes(appToMount))
```

Comment:
Another small change to remove IE support - using Array.prototype.includes rather than Array.prototype.indexOf